hphp/system/php.txt

```diff
@@ -1,5 +1,6 @@
 # Files to include in systemlib.php
 
+
```

Comment:
"This is a mixed ascii/binary" <- garbage.dict is the binary, this is the ascii.